src/Pimple/Container.php

```diff
@@ -279,4 +294,17 @@ public function register(ServiceProviderInterface $provider, array $values = arr
 
         return $this;
     }
+
+    /**
+     * @param $alias
+     * @param $service
+     *
+     * @return $this
+     */
+    public function alias($alias, $service)
+    {
+        $this->aliases[$alias] = $service;
```

Comment:
this needs to check for conflicts with normal keys too.

and aliases are not returned in ``keys()`` wich is also wrong